Modules/Core/SpatialObjects/include/itkSpatialObjectProperty.h

```diff
@@ -74,21 +80,41 @@ class ITKSpatialObjects_EXPORT SpatialObjectProperty
   SetRed(double r);
   double
   GetRed() const;
+  double
+  Red() const
+  {
+    return GetRed();
+  }
```

Comment:
`Red()` looks redundant to me. Doesn't the existing `GetRed()` get "swigged" properly?